src/psyclone/psyGen.py

```diff
@@ -3260,6 +3260,64 @@ def __init__(self, parent=None, variable_name="", valid_loop_types=None,
         self._variable_name = variable_name
         self._id = ""
 
+    @staticmethod
+    def create(var_name, start, stop, step, children):
+        '''Create a Loop instance given valid instances of a variable name,
+        start, stop and step nodes, and a list of child nodes for the
+        loop body.
+
+        :param str var_name: the PSyIR node containing the variable \
+            name of the loop iterator.
+
```

Comment:
Delete this empty line between docstring 'param' and I use a consistent indentation for the 'param' second lines.

Reply:
Oops.